### PR TITLE
fix: typo notes

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change

Change code notes and the ```config.clientQPS ``` judge logic.

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by yanggangtony** [comment]
Hello.
How to re-trigger the ci process? If my branch code have no changed, and the re-base have no function.
Is there a trigger-command to process?
Because the CI: ```Pull Request Changelog Check / Run Changelog Check (pull_request) Failing after 5s``` should re-trigger it .

File: pkg/install/install.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func DaemonSetIsReady(factory client.DynamicFactory, namespace string) (bool, er
 		}
 
 		// Wait for 5 observations of the daemonset being "ready" to be consistent with our check for
-		// the deployment being ready.
+		// the daemonset being ready.
```
[the remark this answers is not in the document]
Comment (yanggangtony) [reply pkg/install/install.go]: Oh, you says that make sense to me ..
Rebased it.

**Review by ywk253100 — DISMISSED** [review]

**Review by reasonerjt — APPROVED** [review]

**Comment by reasonerjt** [comment]
@yanggangtony Could you resolve the conflict?

**Comment by yanggangtony** [comment]
> @yanggangtony Could you resolve the conflict?

Yes, rebased.

**Pull request closed without merging**